<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Utilisateur</class>
 <widget class="QDialog" name="Utilisateur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>995</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#QPushButton{
background-color = rgb(255, 146, 157);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>975</width>
        <height>616</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QTableView" name="uTabV"/>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Actions sur utilisateurs</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="uLabelIditUtilisateur">
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="uTxtNom">
                <property name="placeholderText">
                 <string>Nom</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="uTxtPrenom">
                <property name="placeholderText">
                 <string>Pr√©nom</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="uLabelAdministrateur">
                <property name="text">
                 <string>Administrateur</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="uCBoxAdministrateur">
                <item>
                 <property name="text">
                  <string>0</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>1</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="uTxtUsername">
                <property name="placeholderText">
                 <string>Username</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="uTxtPassword">
                <property name="placeholderText">
                 <string>Password</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="uBtnViderChamps">
                <property name="text">
                 <string>Vider champs</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QPushButton" name="uBtnAjouter">
                  <property name="text">
                   <string>Ajouter</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="uBtnModifier">
                  <property name="text">
                   <string>Modifier</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="uBtnSupprimer">
                  <property name="text">
                   <string>Supprimer</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="uBtnQuitter">
             <property name="styleSheet">
              <string notr="true">background-color = rgb(255, 146, 157);</string>
             </property>
             <property name="text">
              <string>Quitter</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="uLabelAlerte">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
